README.md

```diff
@@ -47,33 +47,33 @@ it is recommended to get started with patchwork, and then look into git-ssb and
 
 # Start the server with extra log detail
 # Leave this running in its own terminal/window
-sbot server --logging.level=info
+ssb-server server --logging.level=info
 
 # publish a message
-sbot publish --type post --text "My First Post!"
+ssb-server publish --type post --text "My First Post!"
 
 # stream all messages in all feeds, ordered by publish time
-sbot feed
+ssb-server feed
 
 # stream all messages in all feeds, ordered by receive time
-sbot log
+ssb-server log
 
 # stream all messages by one feed, ordered by sequence number
-sbot hist --id $FEED_ID
+ssb-server hist --id $FEED_ID
 ```
 ```js
 // In javascript:
 
 var pull = require('pull-stream')
 var ssbClient = require('ssb-client')
 
-// create a scuttlebot client using default settings
+// create a ssb-server client using default settings
 // (server at localhost:8080, using key found at ~/.ssb/secret)
-ssbClient(function (err, sbot) {
+ssbClient(function (err, ssb-server) {
   if (err) throw err
 
   // publish a message
-  sbot.publish({ type: 'post', text: 'My First Post!' }, function (err, msg) {
+  ssb-server.publish({ type: 'post', text: 'My First Post!' }, function (err, msg) {
```

Comment:
This should be ssbServer. PS: there are other occurrences below

Reply:
Nice catch, thanks!